tutor/templates/apps/openedx/config/partials/auth.yml

```diff
@@ -20,5 +20,6 @@ DATABASES:
       {%- if RUN_MYSQL %}
       charset: "utf8mb4"
       {%- endif %}
+      {{ patch("mysql-options") }}
```

Comment:
Let's not add anything to this file, which I would very much like to see disappear. MySQL settings can already be overridden from the edx-platform common settings. Let's not add useless patches.